internal/test/integration/README.md

```
@@ -5,6 +5,15 @@ the test suite for gost-dom.
 test-app/ # The actual HTML server implementation as an http.Handler
-test-app-main/ # A runnable executable that exposes the app on a TCP port 
+test-app-main/ # An app with a real server for inspecting in a browser.
 ```
 
+## Simple dev mode
+
+The `dev` script will start the test server, and restart when a source file is
+saved. The script uses `nodemon` from the NPM ecosystem. You can install it if
+you have node.js and NPM installed using:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)